server/services/curiousService.test.ts

```diff
@@ -148,7 +148,7 @@ describe('curiousService', () => {
             assessmentDate: moment('2012-02-16').utc().toDate(),
             grade: 'Level 1',
             prisonId: 'MDI',
-            prisonName: 'MOORLAND (HMP & YOI)',
+            prisonName: 'Moorland (HMP & YOI)',
```

Comment:
Why this change? 🤔

Reply:
Reformatted to use the lookup.

Reply:
I'm not sure I understand why though? 🤔 
Lets have a quick chat when you get online 👍